crates/compiler/can/src/desugar.rs

```diff
@@ -1893,7 +1814,7 @@ fn desugar_dbg_expr<'a>(
     let region = expr.region;
 
     // tmpVar = expr
-    let ident = &*arena.alloc(var_store.fresh().index().to_string());
+    let ident = arena.alloc(scope.gen_unique_symbol_name().to_string());
```

Comment:
Here's the one line in `desugar.rs` that does more than change `var_store` to `scope`